ui/components/Flex.tsx

```diff
@@ -23,7 +24,8 @@ type Props = {
 const Styled = (component) => styled(component)`
   display: flex;
   flex-direction: ${({ column }) => (column ? "column" : "row")};
-  align-items: ${({ align }) => (align ? "center" : "start")};
+  align-items: ${({ align, alignItems }) =>
+    alignItems ? alignItems : align ? "center" : "start"};
```

Comment:
this is kind of confusing for me to understand - is there a way to write it without the chained ternary?

Reply:
Yes, by using only the `alignItems` props instead of `align` but doing that will require to loop over the Components that uses Flex center and change it so that i didn't change it for now